packages/dev/docs/pages/blog/rtl-date-time.mdx

```diff
@@ -0,0 +1,117 @@
+
+
+```tsx example render=false
+// An example of a date field in ar-AE
+<div>
+  <span aria-label="day">3</span>
+  <span>&rlm;/</span>
+  <span aria-label="month">11</span>
+  <span>&rlm;/</span>  
+  <span aria-label="year">2020</span>
+</div>
+```
+
+```tsx example render=false
+// An example of a date field in he-IL
+<div>
+  <span aria-label="day">3</span>
+  <span>.</span>
+  <span aria-label="month">11</span>
+  <span>.</span>  
+  <span aria-label="year">2020</span>
+</div>
+```
+
+Through much trial and error, we discovered that appplying the [left-to-right embedding (LRE) Unicode](https://unicode.org/reports/tr9/#Explicit_Directional_Embeddings) on the date segments allowed us to to treat the text as embedded left-to-right while preserving the right-to-left mark on the separators, ensuring that Arabic dates display in the correct format. While we could have added Unicode to the segments like we did with the time fields, we opted for the [equivalent CSS](https://unicode.org/reports/tr9/#Markup_And_Formatting) approach instead to avoid modifying the DOM. This CSS was applied on date segments with placeholder or actual values to avoid the behavior discussed earlier with shifting segments. Through additional testing, we found that we should only apply left-to-right embedding on numeric values. If the value was displayed as text, say "November" instead of "11", we did not apply this CSS. 
+
+## Keyboard Navigation
+
+After fixing the formatting though, we also needed to update the keyboard navigation. Previously, if pressing the left arrow key, you would go to the next node in the DOM and vice versa for the right arrow key. After these changes though, visually adjacent elements were not necessarily adjacent in the DOM so this would not work anymore. 
+
+Below is an example of a date field in Hebrew with the correct date format but incorrect keyboard navigation. Pressing the left arrow key should navigate you to the segment to the immediate left, while the right arrow key should navigate you to the segment to your immediate right. 
+
+<Video src={keyboardVideoURL} loop autoPlay muted />
+
+As a result, we updated the keyboard navigation in right-to-left langauges to rely on the positioning of the different segments to determine which node to receive focus for an intuitive keyboard experience.
+
+## Conclusion
+
+If you’ve made it this far, you probably understand just how challenging it is to format dates correctly, especially in right-to-left languages. Fortunately, tools like the Unicode Bidirectional Algorithm help with formatting so we don’t have to handle everything manually. However, as we discovered with this bug, it doesn’t always work as expected and unexpected factors can interfere with it.
```

Comment:
Maybe something like "As you can see, formatting dates correctly is quite challenging, especially in right-to-left languages." for a more confident spin on it?